docs/source/en/task_summary.mdx

```diff
@@ -10,1125 +10,224 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
-...     )
-
-...     print(f"Question: {question}")
-...     print(f"Answer: {answer}")
-Question: How many pretrained models are available in 🤗 Transformers?
-Answer: over 32 +
-Question: What does 🤗 Transformers provide?
-Answer: general - purpose architectures
-Question: 🤗 Transformers provides interoperability between which frameworks?
-Answer: tensorflow 2. 0 and pytorch
-```
-</pt>
-<tf>
-```py
->>> from transformers import AutoTokenizer, TFAutoModelForQuestionAnswering
->>> import tensorflow as tf
-
->>> tokenizer = AutoTokenizer.from_pretrained("bert-large-uncased-whole-word-masking-finetuned-squad")
->>> model = TFAutoModelForQuestionAnswering.from_pretrained("bert-large-uncased-whole-word-masking-finetuned-squad")
-
->>> text = r"""
-... 🤗 Transformers (formerly known as pytorch-transformers and pytorch-pretrained-bert) provides general-purpose
-... architectures (BERT, GPT-2, RoBERTa, XLM, DistilBert, XLNet…) for Natural Language Understanding (NLU) and Natural
-... Language Generation (NLG) with over 32+ pretrained models in 100+ languages and deep interoperability between
-... TensorFlow 2.0 and PyTorch.
-... """
-
->>> questions = [
-...     "How many pretrained models are available in 🤗 Transformers?",
-...     "What does 🤗 Transformers provide?",
-...     "🤗 Transformers provides interoperability between which frameworks?",
-... ]
-
->>> for question in questions:
-...     inputs = tokenizer(question, text, add_special_tokens=True, return_tensors="tf")
-...     input_ids = inputs["input_ids"].numpy()[0]
-
-...     outputs = model(inputs)
-...     answer_start_scores = outputs.start_logits
-...     answer_end_scores = outputs.end_logits
-
-...     # Get the most likely beginning of answer with the argmax of the score
-...     answer_start = tf.argmax(answer_start_scores, axis=1).numpy()[0]
-...     # Get the most likely end of answer with the argmax of the score
-...     answer_end = tf.argmax(answer_end_scores, axis=1).numpy()[0] + 1
-
-...     answer = tokenizer.convert_tokens_to_string(
-...         tokenizer.convert_ids_to_tokens(input_ids[answer_start:answer_end])
-...     )
-
-...     print(f"Question: {question}")
-...     print(f"Answer: {answer}")
-Question: How many pretrained models are available in 🤗 Transformers?
-Answer: over 32 +
-Question: What does 🤗 Transformers provide?
-Answer: general - purpose architectures
-Question: 🤗 Transformers provides interoperability between which frameworks?
-Answer: tensorflow 2. 0 and pytorch
-```
-</tf>
-</frameworkcontent>
+## Computer vision
 
-## Language Modeling
+One of the first and earliest successful computer vision applications was recognizing zip code numbers using a [convolutional neural network (CNN)](glossary#convolution). For many years, CNNs have been the go-to for computer vision tasks, and modern CNNs like [ConvNeXT](./model_doc/convnext) continue to see strong usage with competitive performance.
```

Comment:
Same here. This is the summary of tasks, not the summary of models. I don't think the user comes to this guide to get a history lesson/comparative guide on architectures.